var/spack/repos/builtin/packages/py-cherrypy/package.py

```diff
@@ -0,0 +1,18 @@
+from spack import *
+
+class PyCherrypy(PythonPackage):
+    """CherryPy is a pythonic, object-oriented HTTP framework."""
+
+    homepage = "https://cherrypy.org/"
+    url      = "https://files.pythonhosted.org/packages/1f/de/3327bd7168be762180924085fecef2e127d128f1d6157f88cd87fdac2971/CherryPy-18.1.1.tar.gz"
+
+    version('18.1.1', '76e5d3c6b7be845345f871c604cfdf58')
+
```

Comment:
Missing dep on Python 3.5+

Reply:
Is there some way to say that 1.8 requires 3.5+, but 1.7 doesn't?

Reply:
Found it...